### DISCUSSION TITLE
I can not use coplit with version 1.254.0

### DISCUSSION THREAD

**Discussion by xupeng-mingtu** [community/community#148155] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi Github teams,

I am a paid user of Copilot. After I updated the Copilot plugin to version 1.254.0, I can no longer use Copilot. It prompts me that `It appears you are not eligible to sign up for GitHub Copilot Free.` I have to roll back to 1.246.0 to continue using it. How should I solve this problem?

**Comment by Akash1134** [comment, marked as answer]
Hi @xupeng-mingtu, The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by xupeng-mingtu** [reply]
Thanks for reply!
